doc/user/content/free-trial-faqs.md

```diff
@@ -13,7 +13,7 @@ answers some frequently asked questions about free trials.
 
 ## What are the limits of a free trial?
 
-In Materialize, [clusters](/get-started/key-concepts/#clusters) are the pools of
```

Comment:
If you alias the individual pages, you don't need to replace all the links!

Reply:
Got it.  Just like cleaning up old references for housekeeping purposes even with the alias redirects.   Helps get eyes on other pages, even if just a brief look.